HW03 - Nabilah Sammudin.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "HW03 - Chicken Cochlea Infection Graph"
+author: "Nabilah Sammudin"
+date: "7/20/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+For HW03 assignment, I decided to recreate a figure from my lab's recent publication. I contributed in the execution of the experiments and some of the data analysis but not in the generation of figures. I initially wanted to do both figure A and B from this section but I am having troubles plotting the graph so I might just end up with only Figure A for this assignment. 
+
+I was having troubles plotting repetitive values with ggplot. The graph I initially got with geom_point() looked wrong and that is when I found out that geom_point removes redundancy. I then wondered if character = factor in an Excel file and if I can solve this problem by changing the class_types of my two variables. That didn't work out. I also tried geom_jitters() but the values are all over the place. Technically, the infection variable could also be considered as categorical as it only has four values (0,1,2 and 3). However, plotting two categorical variables would not give me a scatter plot. I settled with using geom_count although it is not exactly what I am looking to recreate. 
+
+There are four sheets in my Excel file. Sheets 1 and 2 are the original data file I got from my lab. For ease of graphing, I had to rearrange/reorder the original data, thus Sheets 3 and 4 are the result of that rearrangement. Sheet 3 which you will see in my codes below is for Graph 1. I spent a really long time on Graph 1 so I wasn't able to get to Sheet 4 for Graph 2.
+
+Here is the image that contains both Figure A and B for reference. 
+
+![Image Link](https://anatomypubs.onlinelibrary.wiley.com/cms/asset/f6fba584-891e-4dc5-b38c-90bff40ebccb/dvdy176-fig-0005-m.jpg
+
+```{r}
+library(readxl)
+
+library("ggplot2")
+excel_sheets("infection.xlsx")
+infection_level <- read_excel("infection.xlsx", sheet = "Rearranged scores (Graph 1)")
+
+age_reorder <- factor(infection_level$age, level = c("E4", "E5", "E6", "E7", "E8", "E9", "E10", "E11", "E12", "E13"))
+ggplot(infection_level, aes(age_reorder, infection)) +
```

Comment:
Hey Nabilah, it's really cool that you used your own published data for this assignment and I enjoyed reading your thought process throughout the code! I liked how you created multiple objects then called them consecutively (infection_level -> age_reorder -> actual plot) to get what you were looking for. I'm hoping we'll soon learn how to use code to organize data so you won't have to manually rearrange/reorder your data in the future!

Reply:
Hi Ben, thank you for your feedback! Yessss, I hope for the same too. I thought about calling the dplyr package to select and filter variables but just the way how the original data was positioned in Excel, it was impossible to apply that package.